src/OrderCompleted/OrderCompleted.js

```diff
@@ -1,3 +1,68 @@
+import styled from 'styled-components'
+import {getCustomerDetail} from '../common/customerInfo'
+import {getCartItems} from '../common/pokemonStorage'
+
+const Wrapper = styled.div`
+  margin: 5px;
+  width: 550px;
+  background-color: #ffffcc ;
+`;
+const Message = styled.h4`
+  padding: 10px;
+  display: flex;
+`;
+const ScrollStyle = styled.div`
+  overflow-y: scroll;
+  max-height: 500px;
+  width: 500px;
+`;
+const CardStyle = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr;
+  img {
+    width: 100px;
+    background-color: cream;
+  }
+  div {
+    padding: 1rem;
+  }
+`;
+
+const CardPriceStyle = styled.div`
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+`;
+const Card = ({name, img, type, price, quantity}) =>(
```

Comment:
should not this `Card` component come from `"./components/Card";` same as in ShoppingCart?